_tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: default
+title: Remote Store Stats API 
+nav_order: 20
+parent: Remote-backed storage
+grand_parent: Availability and recovery
+---
+
+# Remote Store Stats API
+Introduced 2.8
+{: .label .label-purple }
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/1968).
+{: .warning}
+
+Use the Remote Store Stats API to monitor shard-level remote store performance.
+
+## Path and HTTP methods
+
+```json
+GET _remotestore/stats/<index_name>
+GET _remotestore/stats/<index_name>/<shard_id>
+```
+
+## Path parameters
+
+The following table lists the available path parameters. All path parameters are optional.
+
+Parameter | Type | Description
+:--- | :--- | :---
+`index_name` | String | The index name.
+`shard_id` | String | The shard ID.
+
+## Index stats
```

Comment:
Should this be something more parallel with the other heading? For example, "Index stats for all shards" and the other one "Index stats for a single shard"? Or "Get stats for all shards" and "Get stats for single shard"?

Reply:
Changed but I prefer the old one. Let's ask @natebower's opinion.